src/main/kotlin/no/nav/dagpenger/regel/api/arena/adapter/v1/models/digidag/MinsteinntektSubsumsjon.kt

```diff
@@ -11,7 +9,7 @@ data class MinsteinntektSubsumsjon(
     val utfort: LocalDateTime,
     val faktum: MinsteinntektFaktum,
     val resultat: MinsteinntektResultat,
-    val inntekt: Set<Inntekt>
+    val inntekt: List<Inntekt>
```

Comment:
mm, det var egentlig et bevisst valg at det det var et `Set` med inntekt. Et `Set` kan du ikke ha to av samme ting i , men det kan du i en `List` feks.